packages/wonder-blocks-tooltip/src/components/__tests__/tooltip-anchor.test.js

```diff
@@ -113,122 +111,116 @@ describe("TooltipAnchor", () => {
     });
 
     describe("forceAnchorFocusivity is true", () => {
-        test("if not set, sets tabindex on anchor target", async () => {
+        test("if not set, sets tabindex on anchor target", () => {
             // Arrange
-            const ref = await new Promise((resolve) => {
-                const nodes = (
-                    <TooltipAnchor
-                        forceAnchorFocusivity={true}
-                        anchorRef={resolve}
-                        onActiveChanged={() => {}}
-                    >
-                        <View id="portal">This is the anchor</View>
-                    </TooltipAnchor>
-                );
-                mount(nodes);
-            });
+            const nodes = (
+                <TooltipAnchor
+                    forceAnchorFocusivity={true}
+                    anchorRef={jest.fn()}
+                    onActiveChanged={() => {}}
+                >
+                    <View id="portal">This is the anchor</View>
+                </TooltipAnchor>
+            );
+            render(nodes);
 
             // Act
-            const result = ref?.getAttribute("tabindex");
+            const result = screen.getByText("This is the anchor");
 
             // Assert
-            expect(result).toBe("0");
+            expect(result).toHaveAttribute("tabindex", "0");
         });
 
-        test("if tabindex already set, leaves it as-is", async () => {
+        test("if tabindex already set, leaves it as-is", () => {
             // Arrange
-            const ref = await new Promise((resolve) => {
-                const nodes = (
-                    <TooltipAnchor
-                        forceAnchorFocusivity={true}
-                        anchorRef={resolve}
-                        onActiveChanged={() => {}}
-                    >
-                        <View tabIndex={-1}>This is the anchor</View>
-                    </TooltipAnchor>
-                );
-                mount(nodes);
-            });
+            const nodes = (
+                <TooltipAnchor
+                    forceAnchorFocusivity={true}
+                    anchorRef={jest.fn()}
```

Comment:
**question:** are there any tests that check that we set the anchor ref correctly?

Reply:
oh good point, I'll add one!